lib/presentation/home/bloc/home_bloc.dart

```diff
@@ -77,6 +81,11 @@ class HomeBloc extends Bloc<HomeEvent, HomeState> with StreamSubscriptions {
     }
   }
 
+  Future<void> _onTapClearLolPathHomeEvent(TapClearLolPathHomeEvent event, Emitter<HomeState> emit) async {
+    _lcuStore.clear();
+    add(StartHomeEvent());
```

Comment:
We already know state when no path specified, so 
`emit(LolPathUnspecifiedHomeState());`